src/AutoFixture.AutoEF.Tests/Interception/SetPropertyReturnValueInterceptorTest.cs

```diff
@@ -42,6 +42,24 @@ public void SetsPropertyOnValueReturn(SUT sut, IInvocation invocation)
 
             // Verify outcome
             foo.Bar.Should().Be(bar);
+        }
+
+        [Theory, AutoNSub]
+        public void SetsPropertyOnValueReturnWhenTableNameId(SUT sut, IInvocation invocation)
+        {
+            // Setup fixture
+            var far = new Far();
+            var boo = new Boo();
+            invocation.Method.Returns(far.GetType().GetProperty("Boo").GetGetMethod());
+
+            invocation.InvocationTarget.Returns(far);
+            invocation.ReturnValue = boo;
+
+            // Exercise system
+            sut.Intercept(invocation);
+
+            // Verify outcome
+            far.Boo.Should().Be(boo);

```

Comment:
I'm not sure these tests need to exist - it looks like the only changed functionality is in `IdPropertySetterInterceptor` and `ParentPropertySetterInterceptor`, in which case the `SetPropertyReturnValueInterceptor` will not have been touched